tensorboard/webapp/feature_flag/views/types.ts

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
```

Comment:
I'd recommend deferring this particular file to a future PR where you actually introduce the view. Doesn't seem like the types in this file are used by any of the other files in this PR. Its difficult to judge them without any additional context.